alf/drivers/on_policy_driver.py

```diff
@@ -231,9 +231,10 @@ def create_ta(s):
                 time_step, policy_state)
```

Comment:
The returned time_step should also be transformed_time_step. And when final_training_info is created below, the time_step should be transformed_time_step to make the two branches consistent.

Reply:
fixed